App/Components/CafesList.tsx

```diff
@@ -1,97 +1,34 @@
 import React, { Component } from 'react'
-import { Dispatch } from 'redux'
-import { connect } from 'react-redux'
 import { FlatList } from 'react-native'
 import Config from 'react-native-config'
-
-import { RootState, RootAction } from '../Redux/Types'
-import { cafesActions } from '../features/cafes'
+import { Buffer } from 'buffer'
 
 import CafeItem from './CafeItem'
 import Separator from './Separator'
-import { DiscoveredCafes } from '../Services/textile-lb-api'
 
 interface Cafe {
   name: string
   peerId: string
-  url: string
   token: string
 }
 
-const Cafes = [
-  {
-    name: 'US-West-1a',
-    peerId: '12D3KooWGBW3LfzypK3zgV4QxdPyUm3aEuwBDMKRRpCPm9FrJvar',
-    url: 'https://us-west-1a.textile.cafe/',
-    token: Config.RN_TEXTILE_CAFE_TOKEN
-  },
-  {
-    name: 'US-West-1c',
-    peerId: '12D3KooWQue2dSRqnZTVvikoxorZQ5Qyyug3hV65rYnWYpYsNMRE',
-    url: 'https://us-west-1c.textile.cafe',
-    token: Config.RN_TEXTILE_CAFE_TOKEN
-  },
-  {
-    name: 'US-East-2a',
-    peerId: '12D3KooWERmHT6g4YkrPBTmhfDLjfi8b662vFCfvBXqzcdkPGQn1',
-    url: 'https://us-east-2a.textile.cafe',
-    token: Config.RN_TEXTILE_CAFE_TOKEN
-  },
-  {
-    name: 'US-East-2b',
-    peerId: '12D3KooWLh9Gd4C3knv4XqCyCuaNddfEoSLXgekVJzRyC5vsjv5d',
-    url: 'https://us-east-2b.textile.cafe',
-    token: Config.RN_TEXTILE_CAFE_TOKEN
-  },
-  {
-    name: 'EU-West-3a',
-    peerId: '12D3KooWDhSfXZCBVAK6SNQu7h6mfGCBJtjMS44PW5YA5YCjVmjB',
-    url: 'https://eu-west-3a.textile.cafe',
-    token: Config.RN_TEXTILE_CAFE_TOKEN
-  },
-  {
-    name: 'EU-West-3b',
-    peerId: '12D3KooWBCZEDkZ2VxdNYKLLUACWbXMvW9SpVbbvoFR9CtH4qJv9',
-    url: 'https://eu-west-3b.textile.cafe',
-    token: Config.RN_TEXTILE_CAFE_TOKEN
-  },
-  {
-    name: 'AP-Southeast-1a',
-    peerId: '12D3KooWQ5MR9Ugz9HkVU3fYFbiWbQR4jxKJB66JoSY7nP5ShsqQ',
-    url: 'https://ap-southeast-1a.textile.cafe',
-    token: Config.RN_TEXTILE_CAFE_TOKEN
-  },
-  {
-    name: 'AP-Southeast-1b',
-    peerId: '12D3KooWDWJ473M3fXMEcajbaGtqgr6i6SvDdh5Ru9i5ZzoJ9Qy8',
-    url: 'https://ap-southeast-1b.textile.cafe',
-    token: Config.RN_TEXTILE_CAFE_TOKEN
-  }
-]
-
-interface OwnProps {
+interface Props {
   selected?: string
   onSelect: (peerId: string, token: string) => void
   alreadyRegistered?: ReadonlyArray<string>
   ListHeaderComponent?: JSX.Element
   disabled?: boolean
 }
 
-interface StateProps {
-  recommended?: DiscoveredCafes
-}
-
-interface DispatchProps {
-  requestRecommendedCafes: () => void
-}
+const cafesBase64 = Config.RN_TEXTILE_CAFES_JSON
```

Comment:
This value is read from the CircleCI "context" (both beta and production). It is a base64 encoded version of a json data structure that is represented by `Cafe[]`. I've stored the current valid json in 1Password as `production-cafes.json`

Reply:
Sweet. Does the beta app just have one value then?

Reply:
Currently, it's the same for prod and beta and you can manually enter the info for beta. But, I should make them different yea?

Reply:
I think so yeah

Reply:
Ok doing that

Reply:
Ok the beta release will list dev and beta cafes